kubernetes/charts/edge-kubernetes/templates/iotedged-pvc.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if .Values.iotedged.data.persistentVolumeClaim }}
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{ .Values.iotedged.data.persistentVolumeClaim.name | quote }}
+  namespace: {{ include "edge-kubernetes.namespace" . | quote }}
+spec:
+  resources:
+    requests:
+      storage: 100Mi
+  accessModes:
+    - ReadWriteMany
```

Comment:
@darobs Do you remember why it was `ReadWriteMany` for public preview?

Reply:
...because we didn't know any better? :) ...or maybe overly cautious.  Iotedged should be the only pod to read or write to this PVC, so we should make it ReadWriteOnce until we have a reason to do otherwise.